test/main/deepEqual.js

```diff
@@ -1301,12 +1301,24 @@ QUnit.test( "Instances", function( assert ) {
 	b1 = new B();
 	b2 = new B();
 
+	function C( fn ) {
+		this.fn = fn;
+	}
+
+	c1 = new C( function() { return true; } );
+	c2 = new C( function() { return false; } );
+	c3 = new C( function() { return true; } );
+
 	assert.equal( QUnit.equiv( a1, a2 ), true, "Same property, same constructor" );
 
 	// b1.fn and b2.fn are functions but they are different references
 	// But we decided to skip function for instances.
 	assert.equal( QUnit.equiv( b1, b2 ), true, "Same property, same constructor" );
 
+	// Added as bugfix for same property, different reference functions.
+	assert.equal( QUnit.equiv( c1, c2 ), false, "Same property differnent reference, but different functions" );
+	assert.equal( QUnit.equiv( c1, c3 ), true, "Same property different reference, same functions" );
```

Comment:
I would prefer the assertion messages be formatted like such:

* "Same property with different reference and different function"
* "Same property with different reference but same function"

I think it makes it a bit clearer.